libvast/src/system/pivoter.cpp

```diff
@@ -78,6 +78,15 @@ pivoter_state::pivoter_state(caf::event_based_actor*) {
 
 caf::behavior pivoter(caf::stateful_actor<pivoter_state>* self, node_actor node,
                       std::string target, expression expr) {
+  auto normalized_expr = normalize_and_validate(std::move(expr));
+  if (!normalized_expr) {
+    self->quit(caf::make_error(ec::format_error,
+                               fmt::format("pivoter failed to normalize and "
```

Comment:
Generally you can replace the actor name with `{}` and then provide `*self` to the formatter to avoid the duplication. This also applies to the other two instances of this.

Reply:
That's handy. Implemented that in https://github.com/tenzir/vast/pull/2850/commits/acd9d338f87347b0acbaf74216b25a8a3d070460